public/js/lib/devtools/client/shared/components/reps/window.js

```diff
@@ -37,8 +37,8 @@ define(function (require, exports, module) {
       return "";
     },
 
-    getLocation: function (grip) {
-      return getURLDisplayString(grip.preview.url);
+    getDisplayValue: function (grip) {
+      return grip.preview.isGlobal ? "Global" : "Window";
```

Comment:
Just updated the Window rep to use `tiny` mode, and handled attaching the `isGlobal` property by cloning the rep value instead of mutating it.

I didn't realize `kind = ObjectWithUrl` was getting routed correctly to the `Window` rep component, so I also removed the kind change.

I'm still not sure I entirely understand the implications of the `else if (type === "object")` block where the `isGlobal` prop gets added, but it seems to work.